translations/expansion_content.tex/es.po

```diff
@@ -84,7 +84,8 @@ msgstr ""
 #. type: Plain text
 #: sections/expansion_content.tex:3
 msgid "\\addsection{Expansion Content}{\\skills/pathfinding.png}"
-msgstr ""
+msgstr "\\addsection{Ampliación de Contenido}{\\skills/pathfinding.png}\n"
```

Comment:
Argh, this will be a pain to fix...

Reply:
Yes, I saw it... Is possible do it in 2 rows?

Reply:
Changed, better now @qwrtln